### PR TITLE
Improvements - Multiple minor issues

### PR DESCRIPTION
**Pull request by memoriesadrift**

### CODE REVIEW

**Comment by memoriesadrift** [comment]
How much information do we want to display in the search result? See #45

**Comment by michalrzak** [comment]
Hmmmm I pulled it and it really does not look too good. Could you try adding a second row and putting the ects there? We can reduce the fontsize of this second row for it to take up less space.

**Comment by michalrzak** [comment]
BTW please also delete /data/course.dart when deleting the `test course screen` button. We won't need it if the button is no longer there.

File: lib/widgets/study_module_widget.dart [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/material.dart';
+import 'package:we_find/model/model_wrapped.dart';
+import 'package:we_find/screens/study_module_screen.dart';
+
+class StudyModuleWidget extends StatelessWidget {
```
Comment (michalrzak) [inline lib/widgets/study_module_widget.dart]: I think this is an artifact. This class (file) is no longer used and is not contained in the main branch.

File: lib/widgets/favourite_tile.dart [hunk 2]
```diff
@@ -0,0 +1,99 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:we_find/model/model_wrapped.dart';
+import 'package:we_find/providers/fav_course_provider.dart';
+import 'package:we_find/screens/course_detail_screen.dart';
+
+import '../model/model.dart';
```
Comment (michalrzak) [inline lib/widgets/favourite_tile.dart]: nitpick but `import 'package:we_find/model/model.dart' ` is more consistent with the rest of the imports

**Pull request merged**